civihr_default_theme/assets/js/civihr_default_theme.script.js

```diff
@@ -3,53 +3,71 @@
  * Custom scripts for theme.
  */
 
-(function($) {
+(function ($) {
   Drupal.behaviors.civihr_theme = {
-    attach: function (context, settings) {
-
+    attach: function () {
       // on doc ready
       $(document).ready(function () {
+        Drupal.civihr_theme.addClassToRadioButtons();
         Drupal.civihr_theme.applyMatchHeight();
         Drupal.civihr_theme.applyCustomSelect();
         Drupal.civihr_theme.initMobileNav();
+        Drupal.civihr_theme.onBoardingWizard();
       });
 
       // on ajax complete (ie: when opening modals)
       $(document).ajaxComplete(function () {
         Drupal.civihr_theme.applyMatchHeight();
         Drupal.civihr_theme.applyCustomSelect();
       });
-
     }
-  }
+  };
 
   // Create theme related functions
   Drupal.civihr_theme = Drupal.civihr_theme || {};
 
-  Drupal.civihr_theme.initMobileNav = function() {
-    $header = $('.chr_header');
-    $mobileLogo = $('.chr_header__brand.chr_brand.chr_header__home-menu');
-    $nav = $header.find('.chr_header__nav');
+  Drupal.civihr_theme.onBoardingWizard = function () {
```

Comment:
I know this whole file is missing function comments, but can you at least add them to the functions you have modified or created?